apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/Overview/Overview.tsx

```diff
@@ -46,35 +50,35 @@ export const Overview: React.FC = () => {
       return
     }
 
-    try {
-      const shouldSubmitCase = workingCase.state === CaseState.DRAFT
-
-      const caseSubmitted = shouldSubmitCase
-        ? await transitionCase(
-            workingCase,
-            CaseTransition.SUBMIT,
-            setWorkingCase,
-          )
-        : workingCase.state !== CaseState.NEW
+    const shouldSubmitCase = workingCase.state === CaseState.DRAFT
 
-      const notificationSent = caseSubmitted
-        ? await sendNotification(
-            workingCase.id,
-            NotificationType.READY_FOR_COURT,
-          )
-        : false
+    const caseSubmitted = shouldSubmitCase
+      ? await transitionCase(
+          workingCase,
+          CaseTransition.SUBMIT,
+          setWorkingCase,
+        ).catch(() => {
```

Comment:
Consider removing this catch as the error is already handled in the useCase hook.